test/perf/perf_test.go

```diff
@@ -30,8 +30,11 @@ func TestPerformance(t *testing.T) {
 	// host metrics should become available at this point
 	metricsService, err := hostAwait.WaitForMetricsService("host-operator-metrics")
 	require.NoError(t, err, "failed while waiting for the 'host-operator-metrics' service")
+	// host metrics should become available again at this point
+	metricsRoute, err := hostAwait.SetupRouteForService(metricsService, "/metrics")
+	require.NoError(t, err, "failed while setting up or waiting for the route to the 'host-operator-metrics' service to be available")
 
-	count := 1000
+	count := 10
```

Comment:
Did you mean to reduce the count to 10? Maybe we need to somehow make this value configurable by environment? I often change this to 10 when developing and then change back to 1000 too.

Reply:
rahhh, once again, this change was not supposed to be part of the PR! I wanted to run a quick test to capture some metrics, actually :(

Reply:
reverted in https://github.com/codeready-toolchain/toolchain-e2e/pull/157/commits/01e294cb1b77b636143209c8dd9e38145c5f8107